hoot-core/src/main/cpp/hoot/core/io/ServicesDbReader.cpp

```diff
@@ -104,7 +136,39 @@ void ServicesDbReader::_addTagsToElement(shared_ptr<Element> element)
 
     if (!ok)
     {
-      LOG_WARN("Error parsing accuracy.");
+      //check if value contian velocity units
+      if (tags.get("accuracy").contains("km/h", Qt::CaseInsensitive)
+          || tags.get("accuracy").contains("kph", Qt::CaseInsensitive)
+          || tags.get("accuracy").contains("kmph", Qt::CaseInsensitive)
+          || tags.get("accuracy").contains("mph", Qt::CaseInsensitive)
+          || tags.get("accuracy").contains("knots", Qt::CaseInsensitive))
```

Comment:
The error:circular and corresponding accuracy tags should only ever store length values, not velocities.